From penberg@cs.helsinki.fi  Wed Nov 12 13:42:29 2008
From: Pekka J Enberg <penberg@cs.helsinki.fi>
Date: Wed, 29 Oct 2008 20:11:13 +0200 (EET)
Subject: Staging: w35und: remove abs() and BIT() macros
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: Pavel Machek <pavel@suse.cz>
Message-ID: <Pine.LNX.4.64.0810292010570.16596@melkki.cs.Helsinki.FI>


From: Pekka Enberg <penberg@cs.helsinki.fi>

We can use the kernel built-in abs() and BIT() macros the just fine.

Acked-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/linux/common.h |    7 -------
 1 file changed, 7 deletions(-)

--- a/drivers/staging/winbond/linux/common.h
+++ b/drivers/staging/winbond/linux/common.h
@@ -48,16 +48,9 @@
 #define STATUS_MEDIA_CONNECT 1
 #define STATUS_MEDIA_DISCONNECT 0
 
-#ifndef BIT
-#define BIT(x)                  (1 << (x))
-#endif
-
 //==================================================================================================
 // Common function definition
 //==================================================================================================
-#ifndef abs
-#define abs(_T)							((_T) < 0 ? -_T : _T)
-#endif
 #define DEBUG_ENABLED
 #define ETH_LENGTH_OF_ADDRESS	6
 #ifdef DEBUG_ENABLED
